themes/sensei-course-theme/templates/index.html

```diff
@@ -0,0 +1 @@
+
```

Comment:
No, this file is needed, because Wordpress decides if it's a block theme by [checking for](https://github.com/WordPress/WordPress/blob/3cd0e25c034850da8bf6a55ea777115024420a08/wp-includes/class-wp-theme.php#L1478) `templates/index.html`.